conda/base/context.py

```diff
@@ -1114,6 +1116,11 @@ def description_map(self):
                 A list of features that are tracked by default. An entry here is similar to
                 adding an entry to the create_default_packages list.
                 """),
+            'repodata_fn': dals("""
+                Specify a filename for repodata fetching. The standard is
+                repodata.json, but you may want to specify something else to
```

Comment:
The phasing here seems in conflict with the implementation.  The default is to try "current_repodata.json" and if that fail fall back to "repodata.json" but the comment seem to imply that "repodata.json" is the default.